specification/dimensions/tags.md

```diff
@@ -0,0 +1,79 @@
+# Tags
+
+The Tags column represents the set of finalized user-defined and/or provider-defined tags assigned to Tag Sources.  Tags commonly add business context to billing data to identify and accurately allocate charges.
+
+A tag is 'finalized' after a single tag value is chosen from a set of corresponding tag values for a single tag key.  Whether a tag key can have multiple, possible values is determined by the Provider.
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and/or provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter user-defined Tag keys or values.
```

Comment:
Should we have something like, "If a provider allows a resource or resource grouping construct to be tagged, the provider MUST include that tag in the billing data"? We've never used the term "scope", but not sure how to generically reference things that we don't talk about in FOCUS.

Reply:
Good catch. I wasn't trying to make a statement about that. I was suggesting this because I know cases where tags are supposed to be there but aren't. I was hoping to get that covered 😅 We could tweak this to account for that...

> All tags applied to purchases, resources, or resource grouping constructs MUST be included in the billing data. If a provider supports selecting a subset of tags for inclusion in the billing data, the provider MAY only include the configured tags.